tools/azure-sdk-tools/devtools_testutils/proxy_docker_startup.py

```diff
@@ -0,0 +1,136 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import json
+import os
+import logging
+import shlex
+import sys
+import time
+from typing import TYPE_CHECKING
+
+import pytest
+import subprocess
+
+if TYPE_CHECKING:
+    from typing import Optional
+
+
+_LOGGER = logging.getLogger()
+
+CONTAINER_NAME = "ambitious_azsdk_test_proxy"
+LINUX_IMAGE_SOURCE_PREFIX = "azsdkengsys.azurecr.io/engsys/testproxy-lin"
+WINDOWS_IMAGE_SOURCE_PREFIX = "azsdkengsys.azurecr.io/engsys/testproxy-win"
```

Comment:
Well, I didn't know there's an image for windows, was using LINUX_IMAGE on my windows machine. And it looks it works well.

Reply:
Yep, that's expected! [This comment](https://github.com/Azure/azure-sdk-for-python/pull/21538/files#diff-d59281f1d615c6b201013a6521648c6a1491359f395213003fa9f848f89b6142R67) explains a bit more. I also use Linux containers for any local Docker-ing